src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores-export-button.component.ts

```diff
@@ -73,11 +95,12 @@ export class ExerciseScoresExportButtonComponent {
      * @param filename The filename the results should be downloaded as.
      * @param keys The column names in the CSV.
      * @param rows The actual data rows in the CSV.
+     * @param fieldSeparator Optional parameter for exporting the CSV file using a custom separator symbol
      * @private
      */
-    private static exportAsCsv(filename: string, keys: string[], rows: ExerciseScoresRow[]) {
+    private static exportAsCsv(filename: string, keys: string[], rows: ExerciseScoresRow[], fieldSeparator?: string) {
```

Comment:
You can also put default values into the list of parameters, so you don't have to use `?? ';'` below.

Reply:
Applied with [f6129d1](https://github.com/ls1intum/Artemis/commit/f6129d1221beabf2d94daafbaeaf858af0303dbb) 👍